public/locales/en/panel-scene.json

```diff
@@ -0,0 +1,118 @@
+{
+  "hooks": {
+    "selected-node-window-title": "Scene: Selected Node"
+  },
+  "globe-layer": {
+    "error": {
+      "no-property-owner-for-uri": "No property found for URI"
+    },
+    "no-information": "No information"
+  },
+  "scene-graph-node-header": {
+    "out-of-timeframe-tooltip": "This node is hidden because the current time is outside its specified time frame. It will not be visible even if enabled."
+  },
+  "scene-graph-node-meta-info": {
+    "info-table": {
+      "identifier": "Identifier",
+      "uri": "URI",
+      "tags" : "Tags",
+      "about": "About",
+      "about-not-found": "No description found",
+      "gui":  "GUI"
+    },
+    "asset-info-title": "Asset Info",
+    "asset-info-table": {
+      "name": "Name",
+      "path": "Path",
+      "author": "Author",
+      "license": "License",
+      "about": "About",
+      "nodes-in-asset": "Nodes in the asset",
+      "url": "URL",
+      "version": "Version"
+    }
+  },
+  "scene-graph-node-more-menu": {
+    "aria-label": "Open node menu",
+    "delete-button": {
+      "cannot-delete-current-focus": "Cannot delete the current focus node",
+      "info": "Remove this scene graph node (and all its child nodes) from the scene"
+    },
+    "delete-confirm-modal": {
+      "title": "Delete node",
+      "are-you-sure": "Are you sure you want to remove the scene graph node:",
```

Comment:
```suggestion
      "are-you-sure": "Are you sure you want to remove the scene graph node:",
```

Baking in the name into the string like with the `{name}` pattern?  The name might not always come at the end in all languages

Reply:
I get your point, but the name is on a completely different line here

![image](https://github.com/user-attachments/assets/4d64c08c-480e-42a2-8da0-33ebec5623fe)

However, the `:` should be taken out of the string and put into the tsx instead